### PR TITLE
Added TrainingView

### PR DESCRIPTION
**Pull request by duchenean**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -28,9 +28,10 @@ export const settings = {
 export const views = {
   ...defaultViews,
   contentTypesViews: {
+    training_class: TrainingView,
     ...defaultViews.contentTypesViews,
     person: PersonView,
-    presentation: PresentationView,
+    presentation: PresentationView
```
Comment (mpeeters) [inline src/config.js]: The comma at the end of the line should remain, she allow to reduce commit diff and code review

File: src/config.js [hunk 2]
```diff
@@ -28,9 +28,10 @@ export const settings = {
 export const views = {
   ...defaultViews,
   contentTypesViews: {
+    training_class: TrainingView,
```
Comment (mpeeters) [inline src/config.js]: This line must be after `...defaultViews.contentTypesViews` and to respect alphabetical order after `    presentation: PresentationView`

**Comment by duchenean** [comment]
Hey @mpeeters 
Thanks a lot for the feedback. I've fixed the issues.

**Review by mpeeters — APPROVED** [review]
LGTM

**Pull request merged**